quixstreams/sinks/community/pubsub.py

```diff
@@ -0,0 +1,79 @@
+import concurrent.futures
+import logging
+from collections import defaultdict
+from typing import Any
+
+try:
+    from google.api_core.future import Future
+    from google.cloud import exceptions as google_exceptions
+    from google.cloud import pubsub_v1
+except ImportError as exc:
+    raise ImportError(
+        f"Package {exc.name} is missing: "
+        'run "pip install quixstreams[pubsub]" to use PubSubSink'
+    ) from exc
+
+from quixstreams.models.types import HeaderValue
+from quixstreams.sinks.base import BaseSink
+
+logger = logging.getLogger(__name__)
+
+TopicPartition = tuple[str, int]
+
+
+class PubSubSinkTopicNotFoundError(Exception):
```

Comment:
Do we need the word `Sink` here?
```suggestion
class PubSubTopicNotFoundError(Exception):
```